lib/server/readMetadata.js

```diff
@@ -365,17 +365,23 @@ function generateMetadataBlog() {
           filePathDateArr[2] +
           'T06:00:00.000Z'
       );
+      // allow easier sorting of blog by providing seconds since epoch
+      metadata.seconds = Math.round(metadata.date.getTime() / 1000);
```

Comment:
How was it sorted before?

The blog that comes installed with `docusaurus-init` seems sorted by date.

<img width="297" alt="screenshot 2018-02-01 14 44 00" src="https://user-images.githubusercontent.com/3757713/35707411-6eb1b788-075e-11e8-92c6-2293bb0eee28.png">

Reply:
It was sorted based on when the file was loaded and not deterministically. I reproduced several times using my own 20 or so blog files with many different dates. This change guarantees it will always output in reverse chronological order (based on the epoch seconds)